c-bindings-gen/src/main.rs

```diff
@@ -0,0 +1,809 @@
+use std::collections::HashMap;
+use std::env;
+use std::fs::File;
+use std::io::{Read, Write};
+use std::path::Path;
+use std::process;
+
+use proc_macro2::{TokenTree, Span};
+
+mod types;
+use types::*;
+
+/// Prints the docs from a given attribute list unless its tagged no export
```

Comment:
Any chance you could add one-line comments like this on each function across both files? It would help with review and future maintenance.

Reply:
Done!